lib/scala/runtime-version-manager/src/main/scala/org/enso/runtimeversionmanager/runner/NativeExecCommand.scala

```diff
@@ -0,0 +1,32 @@
+package org.enso.runtimeversionmanager.runner
+
+import org.enso.cli.OS
+import org.enso.distribution.{DistributionManager, Environment}
+import org.enso.runtimeversionmanager.components.Engine
+
+import java.nio.file.Path
+
+case class NativeExecCommand(executablePath: Path) extends ExecCommand {
+  override def path: String = executablePath.toString
+
+  override def cmdArguments(
+    engine: Engine,
+    jvmSettings: JVMSettings
+  ): Seq[String] =
+    Seq("-Dcom.oracle.graalvm.isaot=true")
```

Comment:
Interesting. Why do we need to provide this argument?
- I would expect it is only useful at image built time
- or that it is provided by default
- btw. [Native Image API offers some checks](https://www.graalvm.org/sdk/javadoc/org/graalvm/nativeimage/ImageInfo.html) like this too

E.g. if we can access `org.graalvm.nativeimage.ImageInfo` then we can query _"properly"_ and even distinguish between build time and runtime time.

Reply:
The property is completely missing during runtime, hence not allowing me to provide different Context configuration.

Reply:
> access `org.graalvm.nativeimage.ImageInfo` ... distinguish between build time and runtime time.

Another incarnation of this problem [found here](https://github.com/enso-org/enso/pull/12443#discussion_r1985989978)